package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "pattern-replace-loader",
-  "version": "1.0.4",
-  "description": "pattern replace loader for webpack 2",
+  "name": "improved-pattern-replace-loader",
+  "version": "1.0.2",
+  "description": "pattern replace loader for webpack 5",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/aganglada/pattern-replace-loader.git"
+    "url": "https://github.com/mathiasscheffe/pattern-replace-loader.git"
   },
   "keywords": [
     "pattern",
     "replace",
     "string",
     "webpack"
   ],
-  "author": "Alejandro Garcia Anglada (aganglada@gmail.com>)",
+  "author": "Mathias Scheffe",
```

Comment:
same here

Reply:
Yes, I have enabled that maintainers can edit the PR. The package.json is only different because I created a fork of it. Just do ignore the change when merging the PR.